src/main/java/group1/mips_simulator/components/RegisterFile.java

```diff
@@ -0,0 +1,8 @@
+package group1.mips_simulator.components;
+import group1.mips_simulator.components.Register;
+
+public class RegisterFile {
```

Comment:
Can you add some documentation to this class? What is a "RegisterFile" class represent? Just a sentence or two describing what it is. 

How is this distinct from the `MipsComputer.registers` vector?

Reply:
can do